Jenkinsfile

```diff
@@ -13,6 +13,14 @@ pipeline {
   }
 
   stages {
+    stage('Validate') {
```

Comment:
The pipeline isn't currently running because this repo isn't connected to Jenkins (I think).

Reply:
Created issue for this https://github.com/cyberark/summon-s3/issues/11